provisioning/image_customization_test.go

```diff
@@ -106,6 +109,7 @@ func TestNewImageCustomizationContainer(t *testing.T) {
 			{Name: "IRONIC_AGENT_IMAGE", Value: "registry.ci.openshift.org/openshift:ironic-agent"},
 			{Name: "REGISTRIES_CONF_PATH", Value: "/etc/containers/registries.conf"},
 			{Name: "IP_OPTIONS", Value: "ip=dhcp"},
+			{Name: "ADDITIONAL_NTP_SERVERS", Value: ""},
```

Comment:
Bonus points for modifying one of the cases to have a non-empty list.

Reply:
Done